src/libusermode/Makefile.am

```diff
@@ -0,0 +1,145 @@
+# If you have received a written license agreement or contract for        #
+# Covered Software stating terms other than these, you may choose to use  #
+# and redistribute Covered Software under those terms instead of these.   #
+#                                                                         #
+# Source is provided to this software because we believe users have a     #
+# right to know exactly what a program is going to do before they run it. #
+# This also allows you to audit the software for security holes.          #
+#                                                                         #
+# Source code also allows you to port DRAKVUF to new platforms, fix bugs, #
+# and add new features.  You are highly encouraged to submit your changes #
+# on https://github.com/tklengyel/drakvuf, or by other methods.           #
+# By sending these changes, it is understood (unless you specify          #
+# otherwise) that you are offering unlimited, non-exclusive right to      #
+# reuse, modify, and relicense the code.  DRAKVUF will always be          #
+# available Open Source, but this is important because the inability to   #
+# relicense code has caused devastating problems for other Free Software  #
+# projects (such as KDE and NASM).                                        #
+# To specify special license conditions of your contributions, just say   #
+# so when you send them.                                                  #
+#                                                                         #
+# This program is distributed in the hope that it will be useful, but     #
+# WITHOUT ANY WARRANTY; without even the implied warranty of              #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the DRAKVUF   #
+# license file for more details (it's in a COPYING file included with     #
+# DRAKVUF, and also available from                                        #
+# https://github.com/tklengyel/drakvuf/COPYING)                           #
+#                                                                         #
+#*************************************************************************#
+
+sources = userhook.cpp userhook.h
+
+AM_CPPFLAGS = -I$(top_srcdir) -I$(top_srcdir)/src -I$(srcdir)
+AM_CPPFLAGS += $(CPPFLAGS)
+AM_CPPFLAGS += $(VMI_CFLAGS)
+AM_CPPFLAGS += $(GLIB_CFLAGS)
+AM_CPPFLAGS += $(JSONC_CFLAGS)
+
+AM_LDFLAGS =  $(LDFLAGS)
+AM_LDFLAGS += $(GLIB_LIBS)
+AM_LDFLAGS += $(VMI_LIBS)
+AM_LDFLAGS += $(JSONC_LIBS)
+
+AM_CXXFLAGS = $(CXXFLAGS)
+if HARDENING
+AM_CXXFLAGS += $(HARDEN_CFLAGS) -DHARDENING
+endif
+
+if SANITIZE
+AM_CXXFLAGS += $(SANITIZE_CFLAGS)
+AM_LDFLAGS += $(SANITIZE_LDFLAGS)
+endif
+
+if DEBUG
+AM_CXXFLAGS += -DDRAKVUF_DEBUG -Werror -Wall -Wextra -g -ggdb3
+AM_CXXFLAGS += -Wno-missing-field-initializers -Wno-unused-parameter
+AM_CXXFLAGS += -Wno-initializer-overrides -Wno-unknown-warning-option
+AM_CXXFLAGS += -Wno-c99-designator -ferror-limit=0 -Wno-reorder-init-list
```

Comment:
You can remove this line, it compiles fine with those error-detection options enabled.

Reply:
removed